app/models/local_authority.rb

```diff
@@ -68,6 +70,27 @@ def set_active
     self.active = active_attributes?
   end
 
+  def set_defaults
+    self.notify_api_key ||= set_notify_api_key
+    self.letter_template_id ||= set_letter_template_id
+  end
+
+  def set_notify_api_key
+    if Bops.env.production?
+      notify_api_key.presence || Rails.configuration.default_notify_api_key
+    else
+      Rails.configuration.notify_letter_api_key
+    end
+  end
```

Comment:
I think your problem is there's two Notify API keys to consider - there's the one for email and then there's the one for sending letters. The sending letters one is only ever a test api key in staging whereas the email one is live in staging so that we can actually send emails.